ipfsspec/core.py

```diff
@@ -15,6 +15,14 @@
 
 
 class IPFSGateway:
+    """
+    Abstrats the IPFS gateway REST API.
+
+    Documentation for the REST API can be found here: [Kubo_RPC_API]_.
+
+    References:
+        .. [Kubo_RPC_API] http://docs.ipfs.tech.ipns.localhost:8080/reference/kubo/rpc/
```

Comment:
I guess as long as we are talking read only, we might want to stick to the [HTTP Gateway API](https://docs.ipfs.tech/reference/http/gateway/) (not the full RPC API). This should always be available on locally running kubo installations, but should work equally well on public gateways as a fallback.